packages/overlay/test/focus-trap.test.js

```diff
@@ -170,326 +79,55 @@ describe('focus-trap', () => {
       tabKeyDown(document.body);
       expect(getFocusedElementIndex()).to.eql(1);
     });
-
-    it('should not throw using Shift Tab on elements with shadow root', async () => {
```

Comment:
This test was added in https://github.com/vaadin/vaadin-overlay/pull/67 and is now obsolete (related logic was changed).